evals/registry/solvers/gemini.yaml

```diff
@@ -21,3 +21,29 @@ generation/cot/gemini-pro:
       class: evals.solvers.providers.google.gemini_solver:GeminiSolver
       args:
         model_name: gemini-pro
+
+generation/direct/gemini-pro-flash-wav:
+  class: evals.solvers.providers.google.gemini_solver_wav:GeminiSolverWav
+  args:
+    model_name: gemini-1.5-flash
```

Comment:
The naming looks confusing to me. Why include both `pro` and `flash` in the solver name: `gemini-pro-flash-wav1`? Same question for the other solvers.

Reply:
would the gemini version `1.5` also be included in the solver name?